engine/runtime-compiler/src/test/java/org/enso/compiler/test/pass/MiniPassTraverserTest.java

```diff
@@ -125,4 +133,32 @@ public void chainedMiniPass_StopsTraversingWhenPrepareFromBothPassesReturnNull()
     assertThat("e1 should be processed by both passes", e1.isTransformedBy(miniPass1), is(true));
     assertThat("e1 should be processed by both passes", e1.isTransformedBy(miniPass2), is(true));
   }
+
+  /** MiniPassTraverser ignores Case.Branch.pattern */
```

Comment:
- Is it OK that the `Case.Branch` is ignored? 
- Or should something be fixed?

Reply:
To be honest, I am not sure about it. At first, I thought that it is a bug, but since all our other tests are passing, and we have been using mini passes for many months now, I guess it is a feature now. At the end, I have given up the idea of `UnusedImports` as a mini pass. At least it is worth having that in the test to understand the semantics better.